src/main/java/org/mockito/internal/progress/MockingProgressImpl.java

```diff
@@ -143,4 +146,18 @@ public void setVerificationStrategy(VerificationStrategy strategy) {
     public VerificationMode maybeVerifyLazily(VerificationMode mode) {
         return this.verificationStrategy.maybeVerifyLazily(mode);
     }
-}
+
+    public void setStubbingListener(StubbingListener stubbingListener) {
+        this.stubbingListener = stubbingListener;
+    }
+
+    public StubbingListener getStubbingListener() {
+        if (this.stubbingListener == null) {
+            return new StubbingListener() {
```

Comment:
Actually, there might be a case of multiple listeners being attached (just like with observers) so lets turn this into a list? Then it can be initialized on the field as empty arraylist and a simple for loop to process all listeners. Later wirh streams it can be enhanced to provide a function callback and turn it into a oneliner.